README.md

```diff
@@ -121,17 +121,11 @@ For a good example package, see [base64-js](https://github.com/DefinitelyTyped/D
 #### Common mistakes
 
 * First, follow advice from the [handbook](http://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html).
-* Formatting: Either use all tabs, or always use 4 spaces. Also, always use semicolons, and use egyptian braces.
-* `interface X {}`: An empty interface is essentially the `{}` type: it places no constraints on an object.
-* `interface IFoo {}`: Don't add `I` to the front of an interface name.
```

Comment:
We lint for it: [interface-name](https://palantir.github.io/tslint/rules/interface-name/) with ["never-prefix"](https://github.com/Microsoft/types-publisher/blob/master/tslint-common.json).